src/components/app/App.js

```diff
@@ -1,8 +1,11 @@
 import Component from '../Component.js';
 import Header from './Header.js';
-import Search from '../options/Search.js';
-import Sort from '../options/Sort.js';
 import PokeDex from '../poke-dex/PokeDex.js';
+import Search from '../options/Search.js';
+import Paging from '../options/Paging.js';
+// import Sort from '../options/Sort.js';
```

Comment:
make sure you delete any dead code, console.logs, and comments before you submit!